plugins/portals-list.js

```diff
@@ -4,6 +4,8 @@
 // @version        0.4.2
 // @description    Display a sortable list of all visible portals with full details about the team, resonators, links, etc.
 
+/* global plugin,DEPLOY_RESONATOR,CAPTURE_PORTAL,COMPLETION_BONUS,DESTROY_RESONATOR,DESTROY_LINK,DESTROY_FIELD -- eslint */
```

Comment:
```suggestion
```

Yeah, really no!

When using these in code use the `window.` prefix.
Then they won't be needed here.